e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -1,6 +1,5 @@
 import _ from "underscore";
 
-import { loginCache } from "e2e/support/commands/user/authentication";
```

Comment:
This has been reverted to save me pain when running Cypress locally. We should unrevert before merging.